dotnet/src/SemanticKernel.UnitTests/Planning/PlanTests.cs

```diff
@@ -855,7 +855,7 @@ void FunctionInvoked(object? sender, FunctionInvokedEventArgs e)
         var result = await sut.RunAsync("PlanInput", plan);
 
         // Assert
-        Assert.NotNull(result);
+        Assert.Null(result);
```

Comment:
This looks strange. Why the flip?

Reply:
KernelResults would be returned with 0 functions.

When the function is the result, I agree don't make sense to return a FunctionResult without result...

Reply:
Agreed offline to keep it this way taking into account that the method is going to be moved to samples. However, it should be fixed in Function.InvokeAsync in scope of this issue - https://github.com/microsoft/semantic-kernel/issues/3586